requirements/docker.txt

```diff
@@ -3,4 +3,5 @@
 
 money-to-prisoners-common[monitoring]>=9.18,<9.19
 
+sentry-sdk==0.16.5
```

Comment:
is this here temporarily before mtp-common is updated?

Reply:
Yup